eng/templates/build-and-test-job-linux-templates.yml

```diff
@@ -0,0 +1,41 @@
+# build-and-test-job-linux-template.yml
+parameters:
+  buildConfig: 'Release'
+  buildArgs: ''
+  skipTests: false
+  additionalArgs: ''       # Additional arguments to pass to the test command
+
+steps:
+- template: /eng/templates/setup-template.yml@self
+
+- script: |
+    mkdir -p "$(Build.SourcesDirectory)/artifacts"
+    cp eng/resources/* "$(Build.SourcesDirectory)/artifacts"
+  displayName: Prevent test directory crawling
+
+- pwsh: |
+    $testArg = if ($env:SKIPTESTS -ne "true") { "--test" } else { "" }
+    Write-Host "##vso[task.setvariable variable=_TestArgs]$testArg"
+  displayName: Promote variables
+
+- script: ./eng/cibuild.sh
```

Comment:
Full path here.

Reply:
Do you mean to use /eng/cibuild.sh  instead of  ./eng/cibuild.sh  ?

Reply:
done. Also added a @self

Reply:
Replaced with /eng/cibuild.sh

Reply:
done